### PR TITLE
Hide header course menu at a breakpoint

### PR DESCRIPTION
**Pull request by capajon**

### CODE REVIEW

File: header/actions.html [hunk 1]
```diff
@@ -29,6 +29,11 @@
 				display: none;
 			}
 		}
+		@media(max-width: 554px) {
```
Comment (dlockhart) [inline header/actions.html]: Don't we want to hide it at the same breakpoint as we show the hamburger menu (992px)?

Comment (dlockhart) [inline header/actions.html]: The hiding of the things that shouldn't be shown when the mobile menu is shown is happening here:
https://github.com/Brightspace/d2l-navigation-ui/blob/master/header/header.html#L54

Comment (capajon) [inline header/actions.html]: We had a lengthy-ish discussion debating that and we wanted to go with this for now... it is the same breakpoint as when the divider disappears in the actions area.

We don't disagree with you - but find that the hamburger menu shows up a bit earlier than we think it needs to, and for now we'd like to keep the course menu showing even though there is some duplication for 400px or so....

Jeff is going to make a story to look at all the breakpoints and it'll be adjusted then.

Comment (dlockhart) [inline header/actions.html]: OK sounds good. Yeah I think all the breakpoints needs to be looked at -- I originally just implemented the design as it was, but for sure some of them are not based on when the content needs to break, but instead just arbitrary pixel values.

**Pull request merged**